tests/unit/models/test_projects.py

```diff
@@ -2505,26 +2505,21 @@ def test_component_name_valid(
     @pytest.mark.parametrize(
         "name,error",
         [
-            (
-                "snap-",
-                "Component names cannot start with the reserved namespace 'snap-'",
-            ),
-            (
+            pytest.param(
```

Comment:
We can test name0123 and 0123name as valid component names above.

Reply:
Added!